packages/react/src/components/DescriptionList/index.test.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import {
+  DescriptionList,
+  DescriptionListItem,
+  DescriptionTerm,
+  DescriptionDetails
+} from './index';
+import axe from '../../axe';
+
+describe('DescriptionList components', () => {
+  test('renders a dl element', () => {
+    const { container } = render(<DescriptionList>a</DescriptionList>);
+    const dl = container.querySelector('dl');
+    expect(dl).toBeInTheDocument();
+  });
+
+  test('renders a dt element', () => {
+    const { container } = render(<DescriptionTerm>a</DescriptionTerm>);
+    const dt = container.querySelector('dt');
+    expect(dt).toBeInTheDocument();
+  });
+
+  test('renders a dd element', () => {
+    const { container } = render(<DescriptionDetails>a</DescriptionDetails>);
+    const dd = container.querySelector('dd');
+    expect(dd).toBeInTheDocument();
+  });
+
+  test('handles uncollapsed prop', () => {
+    const { container } = render(<DescriptionList>a</DescriptionList>);
+    const dl = container.querySelector('dl');
+    expect(dl).not.toHaveClass('DescriptionList--collapsed');
+  });
+
+  test('handles collapsed prop', () => {
+    const { container } = render(
+      <DescriptionList collapsed>a</DescriptionList>
+    );
+    const dl = container.querySelector('dl');
+    expect(dl).toHaveClass('DescriptionList--collapsed');
+  });
+
+  test('passes classNames through', () => {
+    render(<DescriptionList className="a">a</DescriptionList>);
+    const dl = screen.getByText('a');
+    expect(dl).toHaveClass('a');
```

Comment:
We should be sure that we're getting the right component here (by its role or test id) and that it doesn't clobber the existing class name that's already on the component.